src/ert/_c_wrappers/enkf/config/field_config.py

```diff
@@ -34,7 +34,9 @@ class Field(ParameterConfig):
     output_file: Path
     grid_file: str
 
-    def load(self, run_path: Path, real_nr: int, ensemble: EnsembleAccessor):
+    def read_from_runpath(
```

Comment:
This is actually `read_from_runpath_and_save_to_storage`. Perhaps just call it that so it sticks out?

Reply:
After your PR I would really like to split out the `and_save_to_storage` part to, so it becomes: 
`read_from_runpath_and_convert_to_dataset`, but feel it can be `read_from_runpath` in that case

Reply:
Ok, that makes sense